File: app/src/main/java/com/facundoaramayo/testrappikotlin/data/Constant.java
package com.facundoaramayo.testrappikotlin.data;

public class Constant {

    public static String WEB_URL = "https://developers.zomato.com/";

    //TODO: Quitar datos estáticos de geoposicionamiento
    /*Los Ángeles*/
    public static final double city_lat = 33.94;
    public static final double city_lng = -118.41;
    public static final float city_zoom = 10;

    public static String getURLimgPlace(String file_name) {
        return file_name;
    }

    // Este valor límite se utiliza para dar paginación (solicitud y visualización) para disminuir el payload
    public static final int LIMIT_PLACE_REQUEST = 40;
    public static final int LIMIT_LOADMORE = 40;



}

File: app/src/main/java/com/facundoaramayo/testrappikotlin/data/Constant.kt
package com.facundoaramayo.testrappikotlin.data

object Constant {

    var WEB_URL = "https://developers.zomato.com/"

    const val CITY_ZOOM = 10f

    // Este valor límite se utiliza para dar paginación (solicitud y visualización) para disminuir el payload
    const val LIMIT_PLACE_REQUEST = 40
    const val LIMIT_LOADMORE = 40

    fun getURLimgPlace(file_name: String): String {
        return file_name
    }


}
